Ansible/roles/liferay-csgf/tasks/domain.yml

```diff
@@ -4,7 +4,7 @@
 # role_vars
 - name: Copy the Glassfish server password file
   copy:
-    src=passwordfile dest=/opt/glassfish/glassfish3/bin/passwordfile
+    src=passwordfile dest=/home/liferayadmin/passwordfile
```

Comment:
Why is this difference important? What happens when you write straight to that directory ?